about/about.html

```diff
@@ -0,0 +1,126 @@
+
+    <section class="intro-section intro-section-about padding-x text-center">
+      <article class="content content-about">
+        <header class="title">
+          <h1 class="display-1 display-1-about text-secondary">18th Latin American Congress on Free Software and Open
+            Technologies</h1>
+        </header>
+        <p class="par">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Harum aperiam sunt ut esse delectus,
+          aliquam ea reiciendis laudantium maxime aliquid. Sed aliquam harum obcaecati consectetur consequatur soluta
+          nihil eveniet blanditiis, sunt repudiandae excepturi delectus corporis illum natus placeat dignissimos
+          aspernatur? Ex dignissimos nisi aliquam laudantium? Temporibus consequatur tempora laboriosam possimus.
+        </p>
+        <div class="intro-bottom">
+          <p class="par">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Autem cumque molestias voluptate
+            blanditiis placeat rem?</p>
+          <a class="text-primary" href="#">Hello!123@fake.com</a>
+        </div>
+      </article>
+    </section>
+    <!-- ************************************************************************************************************************
+                                                    LOGO SECTION
+    ************************************************************************************************************************ -->
+    <section class="logo-section padding-y padding-x flex text-center">
+      <h1 class="display-3 text-primary">Latin American Congress on Free Software and Open Technologies's Logo</h1>
+      <div class="section-line"></div>
+      <p class="par par-logo">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Nam esse laudantium nulla porro eius
+        optio accusantium? A atque harum itaque.</p>
+      <div class="logo logo-about"></div>
+    </section>
+    <!-- ************************************************************************************************************************
+                                                    SPEAKERS SECTION
+    ************************************************************************************************************************ -->
+    <section class="speakers-section speakers-section-about padding-y padding-x text-center flex">
+      <h1 class="h3 text-primary">Featured Speakers</h1>
+      <div class="section-line"></div>
+      <p class="par"> Lorem ipsum dolor sit amet consectetur, adipisicing elit. Impedit, illo! Possimus, placeat.</p>
+      <article class="image-container flex">
+        <figure class="person-img-wrapper">
+          <img class="congress-img" src="../assets/images/main/congress-1.png" alt="congress-1">
+        </figure>
+        <figure class="person-img-wrapper">
+          <img class="congress-img" src="../assets/images/main/congress-2.png" alt="congress-2">
+        </figure>
+      </article>
+    </section>
```

Comment:
- [ ] Good job making the suggested changes here👍, kindly change the title from featured speaker to something else😉

Reply:
- [ ] Please do make sure you don't have more than one `h1` tag for each page because having multiple make your code less semantic😎😎 and confuse the search engine when seaRCHING FOR YOUR PAGE